libs/server-sdk/src/data_sources/streaming_data_source.cpp

```diff
@@ -0,0 +1,152 @@
+#include <boost/asio/any_io_executor.hpp>
+#include <boost/asio/io_context.hpp>
+#include <boost/asio/post.hpp>
+#include <boost/json.hpp>
+#include <boost/url.hpp>
+
+#include <utility>
+
+#include "streaming_data_source.hpp"
+
+#include <launchdarkly/network/http_requester.hpp>
+
+namespace launchdarkly::server_side::data_sources {
+
+static char const* const kCouldNotParseEndpoint =
+    "Could not parse streaming endpoint URL.";
```

Comment:
```suggestion
    "Could not parse streaming endpoint URL";
```
Just commenting for consistency with the other logs. But we could also chose to terminate them all with periods.